pydantic/validate_call_decorator.py

```diff
@@ -16,6 +17,50 @@
     AnyCallableT = TypeVar('AnyCallableT', bound=Callable[..., Any])
 
 
+def _check_function_type(function: object):
+    ERROR_CODE = 'validate-call-type'
```

Comment:
Can be defined outside of the scope of the function, perhaps with a more descriptive name